plugin/cr/src/main/java/org/opencds/cqf/ruler/cr/CrProperties.java

```diff
@@ -19,6 +20,14 @@ public void setEnabled(boolean enabled) {
 		this.enabled = enabled;
 	}
 
+	public boolean getCareGapsParallelEnabled() {
```

Comment:
Can we use the word threaded rather than parallel? That's to be consistent with the Measure evaluation.